### PR TITLE
Tidied up introduction and made minor changes to formatting

### PR DESCRIPTION
**Pull request by 4katie**
Signed-off-by: Katie <sa951009@gmail.com>

# Issue #111 
Tied up `Introduction` and made changes according to @Michael-Grover 's comments 

### Changes
(Structure)
- Changed the order of documents in the `Introduction` category
- Changed structure slightly, so that it's now `Overview`, `Key Concepts`, `For Business`, and `For Developers` 

(Minor fixes)
- `started-hello.md`, `started-installation.md`: Minor wording changes or formatting changes, following feedback from @Michael-Grover 
- Many documents: changed `[{}]` notation to `{{}}` notation

(Content changes)
- `accordproject`:
  - Removed references to businesses, so that this can be an introduction to the AP for all users 

- `accordproject-concepts`:
  - Changed code snippets for `model` and `logic` (using the new version in https://github.com/accordproject/cicero-template-library/tree/js-release-0.20/src/acceptance-of-delivery) following request from @jeromesimeon 
  - Included references to glossary following request from @peterhunn 

- `accordproject-business`:
  - Now contains both 'for lawyers' and 'for businesses' - may need to rewrite something
  - included link to slack channel 

- `model-relationships.md`: 
   - formatted list at the top
   - changed `Model Manager` to `Concerto`
   - fixed typo 

- `started-studio.md`: included a timestamp of the contents of the video for easier user navigation

- `started-resources.md`: changed hyperlinked text to better reflect link
 
- `ref-glossary.md`: included some more accessible definitions

### Flags
- Hyperlinks in bulletpoints do not show as a different color... 

### Related Issues
- All issues related to improving `v0.20` docs

### CODE REVIEW

File: docs/accordproject-concepts.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ evaluate the {{deliverable}} on the delivery date before notifying
 
 ## Acceptance Criteria.
 
-The "Acceptance Criteria" are the specifications the {{deliverable}}
+The 'Acceptance Criteria' are the specifications the {{deliverable}}
```
Comment (4katie) [inline docs/accordproject-concepts.md]: changed so that the words won't show up as green

File: docs/model-relationships.md [hunk 2]
```diff
@@ -5,21 +5,21 @@ title: Relationships
...
 
 Hence a relationship could be: `org.example.Vehicle#123456`
 
-This would be a relationship to the `Vehicle` type declared in the `org.example` namespace with the identifier `123456`.
+This would be a relationship to the `Vehicle` _type_ declared in the `org.example` _namespace_ with the _identifier_ `123456`.
 
 Relationships are unidirectional and deletes do not cascade, ie. removing the relationship has no impact on the thing that is being pointed to. Removing the thing being pointed to does not invalidate the relationship.
 
-Relationships must be resolved to retrieve an instance of the object being referenced. The act of resolution may result in null, if the object no longer exists or the information in the relationship is invalid. Resolution of relationships is outside of the scope of the Model Manager.
+Relationships must be resolved to retrieve an instance of the object being referenced. The act of resolution may result in null, if the object no longer exists or the information in the relationship is invalid. Resolution of relationships is outside of the scope of Concerto.
```
Comment (4katie) [inline docs/model-relationships.md]: I'm not sure if this was supposed to change or was meant to stay as Model Manager

Comment (4katie) [inline docs/model-relationships.md]: I'm not sure if this was supposed to change or was meant to stay as Model Manager

File: docs/ref-glossary.md [hunk 3]
```diff
@@ -3,4 +3,54 @@ id: ref-glossary
...
+
+The Data Model is used to create a machine-readable representation of the text of the contract. It does this by creating a link with the text of the contract by defining the Variables that should exist within the contract along with an associated data type. The linkage between the text and the data model is created by referencing the variable in both the model and the text.
+
+
+For example, if the text is:
+```
+Upon the signing of this Agreement, {{buyer}} shall pay {{amount}} to {{seller}}.
+```
+
+The model will include `buyer`, `amount`, and `seller`. A data type and a value is assigned against each of these variables.
```
Comment (irmerk) [inline docs/ref-glossary.md]: I'm inclined to think there would be explanation or a section on data types.

File: docs/spec-template.md [hunk 4]
```diff
@@ -3,10 +3,10 @@ id: spec-template
...
 
-An Accord Project template is composed of three elements: 
+An Accord Project template is composed of three elements:
 
 - Natural Language, the grammar for the legal text of the template
-- Model, the data model that backs the template 
+- Model, the data model that backs the template
 - Logic, the executable business logic for the template
 
 ![Cicero Template](assets/template.png)
```
Comment (irmerk) [inline docs/spec-template.md]: If we are changing the formatting to `{{` and `}}`, we should update the diagram too.

**Pull request merged**